client/blocks/eligibility-warnings/style.scss

```diff
@@ -5,6 +5,23 @@
     line-height: 1.5;
 }
 
+.is-placeholder {
```

Comment:
@aduth Created a PR to fix this: https://github.com/Automattic/wp-calypso/pull/10242